co-log/co-log.cabal

```
@@ -97,3 +97,15 @@ executable readme
   build-tool-depends:  markdown-unlit:markdown-unlit
   ghc-options:         -Wall -pgmL markdown-unlit
   default-language:    Haskell2010
+
+test-suite property
```

Just more meaningful name for the `test-suite` stanza.
```suggestion
test-suite test-co-log
```